app/src/main/java/ch/blinkenlights/android/vanilla/Song.java

```diff
@@ -226,6 +228,17 @@ public String getTrackAndDiscNumber() {
 		return result;
 	}
 
+	/**
+	 * @return title of this song, or the filename if the title is not available
+	 */
+	public String getTitle() {
```

Comment:
'title' would need to be set when the object is created: The code here would block the UI thread, which is pretty bad.